cirq/circuits/circuit.py

```diff
@@ -1410,7 +1410,7 @@ def unitary(self,
 
     def final_wavefunction(
             self,
-            initial_state: Union[int, np.ndarray] = 0,
+            initial_state: 'cirq.STATE_VECTOR_LIKE' = 0,
```

Comment:
I mean ... this really exacerbates the "wavefunction vs. state vector" terminology question

Reply:
You have a point.